uk.co.compucorp.civicrm.hrcore/backstop_data/data/onboarding-wizard-data.js

```diff
@@ -0,0 +1,39 @@
+module.exports = (function () {
+  return {
+    personalDetails: {
+      '#edit-submitted-civicrm-1-contact-1-contact-first-name' : 'Duke',
+      '#edit-submitted-civicrm-1-contact-1-contact-last-name' : 'Duke',
+      '#edit-submitted-civicrm-1-contact-1-contact-birth-date' : '2017-10-11',
+      '#edit-submitted-civicrm-1-contact-1-cg5-custom-15' : '1002'
+    },
+    address: {
+      '#edit-submitted-civicrm-1-contact-1-address-street-address' : 'Duke',
+      '#edit-submitted-civicrm-1-contact-1-address-supplemental-address-1' : 'Duke',
+      '#edit-submitted-civicrm-1-contact-1-address-supplemental-address-2' :  'Duke',
+      '#edit-submitted-civicrm-1-contact-1-address-city' : 'Duke',
+      '#edit-submitted-civicrm-1-contact-1-address-postal-code' : 'Duke',
+      '#edit-submitted-civicrm-1-contact-1-address-country-id' : '1226'
+    },
+    contactInfo: {
+      '#edit-submitted-civicrm-1-contact-1-contact-nick-name' : 'Duke'
+    },
+    payroll: {
+      '#edit-submitted-civicrm-1-contact-1-cg4-custom-7' : 'Duke',
+      '#edit-submitted-civicrm-1-contact-1-cg4-custom-8' : 'Duke',
+      '#edit-submitted-civicrm-1-contact-1-cg4-custom-9' : 'Duke',
+      '#edit-submitted-civicrm-1-contact-1-cg4-custom-10' : 'Duke',
+      '#edit-submitted-civicrm-1-contact-1-cg16-custom-72' : 'Duke',
+      '#edit-submitted-civicrm-1-contact-1-cg5-custom-19' : 'single'
+    },
+    emergencyContacts: {
+      '#edit-submitted-emergency-contact-civicrm-1-contact-1-cg99999-custom-100000' : 'Duke',
```

Comment:
These field names look random, is it ok to use them? Isn't there some other selector we could use? Why not use the name field? `[name="emergency_contact"]` (if possible)